best-practices/containers.md

```diff
@@ -0,0 +1,78 @@
+# Guide for DLSS Developers working with containers
+
+The infrastructure team has been utilizing docker containers for local developement,
+particularly when supporting external dependencies for somet time......
+
+Most of this guide will focus on docker compose...
+
+## Docker
+
+### Installing docker
+
+Install docker desktop for your local operating system:
+- [MacOS](https://docs.docker.com/desktop/mac/install/)
+- [Linux](https://docs.docker.com/desktop/linux/install/)
+
+### Working with docker compose
```

Comment:
should this be a level 2 heading with the following items underneath it?

Reply:
I think you're right, that would be better. Thanks.